admin_guide/manage_scc.adoc

```diff
@@ -274,6 +274,24 @@ This allows images to run as the root UID if no *USER* is specified in the
 *_Dockerfile_*.
 ====
 
+[[enable-dockerhub-images-that-require-root]]
+
+=== Enable dockerhub images that require root
+
+Some Dockerhub images (examples: `postgres` and `redis`) require root access and have certain
+expectations about how volumes are owned.  For these images, add the service account to the
+`anyuid` SCC.
```

Comment:
The "anyuid" SCC is one of the new predefined SCCs in version 3,1; however, it is not listed in the 2 predefined SCCs shown in the ```oc get scc``` commands above. I think the outputs of the two ```oc get scc``` commands need to be updated.